pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -46,8 +46,16 @@ type TFJobSpec struct {
 	// CleanPodPolicy defines the policy to kill pods after TFJob is
 	// succeeded.
 	// Default to Running.
+	// TODO: CleanPodPolicy is deprecated. It should be deleted later.
```

Comment:
@gaocegege what's the policy of deprecating this API ? this API is recently added